nekoyume/Assets/_Scripts/Game/Controller/ColorController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using Nekoyume.Pattern;
+using UnityEngine;
+
+namespace Nekoyume.Game.Controller
+{
+    public class ColorController : MonoSingleton<ColorController>
+    {
+        private List<Color> _colors;
+
+        public void Initialize()
+        {
+            var colorRef = Resources.Load<ColorReferenceScriptableObject>("ScriptableObject/UI_ColorReference");
+            _colors = colorRef.datas;
+        }
+
+        public static Color Color(int index)
```

Comment:
How about using colors bound with `enum`?
I think it's hard to directly recognize color with integer value.

Reply:
I used the index for two reasons.
First, the use of enumeration requires mapping.
Second, the naming is ambiguous when one color value is used in several places.
The meaning of the index will be documented in the wiki.

Reply:
I didn't intend to map enum value with color data itself, but with the parameter.
This is an example.
```
public enum PaletteColorType
{
    White = 0,
    ButtonEnabled = 1,
    ButtonDisabled = 2
}
```
And if the parameter type is changed to `PaletteColorType`,
You can pass color like `Color(PaletteColorType.ButtonEnabled)` and convert it into integer in internal operation like this. 
```
public static Color Color(PaletteColorType colorType)
{
    var index = (int) colorType;
    return instance._colors[index] != null
        ? instance._colors[index]
        : UnityEngine.Color.black;
}
```

Reply:
This color data is managed by the designer.
If we use enum, the designer should add enum.
Also, one color value can be used in many places.
For example, ButtonEnabled, Dimmed, etc..

Reply:
Then, how about using string? I still think using integer as color value has so much disadvantage than advantage we can get.

Reply:
I agree that Intex is not good. Then i'll improve this later and proceed with simple refactoring. Please review again:pray: